cpp/src/rolling/rolling.cu

```diff
@@ -14,9 +14,12 @@
  * limitations under the License.
  */
 
-#include <rolling/jit/code/code.h>
-#include <rolling/rolling_detail.hpp>
-#include <rolling/rolling_jit_detail.hpp>
+#include "jit/launcher.h"
+#include "jit/parser.h"
+#include "jit/type.h"
+#include "rolling/jit/code/code.h"
+#include "rolling/rolling_detail.hpp"
+#include "rolling/rolling_jit_detail.hpp"
```

Comment:
Is there a reason these are quoted instead of bracketed now?

Reply:
Yes, IIRC it's because they're in the `src` dir and not the `include` dir. I recall we applied this convention for "private" headers that're under `src`, but lemme know if that's changed.

Reply:
We only use quotes when the path is in the same folder as the present file.  These used to be <>, which was correct, because while there is a src/rolling/jit folder, jit/launcher.h is in src/jit/, which is a different path.  

So this should be:

```suggestion
#include <jit/launcher.h>
#include <jit/parser.h>
#include <jit/type.h>

#include "jit/code/code.h"
#include "rolling_detail.hpp"
#include "rolling_jit_detail.hpp"
```

Also note that the latter three should be moved to the top of the file (as they were before), since they are the "nearest" includes, and we order from nearest to farthest.

(I just cleaned up the includes in nearly every file in the previous release.)